src/tools/compiletest/src/header.rs

```diff
@@ -51,6 +52,14 @@ impl EarlyProps {
         let has_tsan = util::TSAN_SUPPORTED_TARGETS.contains(&&*config.target);
         let has_hwasan = util::HWASAN_SUPPORTED_TARGETS.contains(&&*config.target);
 
+        if config.mode == Mode::RMC {
+            // If the path to the test contains "fixme" or "ignore", skip it.
+            let re = Regex::new(r"fixme|ignore").unwrap();
+            if re.is_match(testfile.to_str().unwrap()) {
```

Comment:
Importing and using regex to check for this seems a bit overkill to me. Can't we just use `contains` on the test filename?

Reply:
We can. Modified the code to use `contains`.